third_party/go/BUILD

```diff
@@ -661,6 +661,7 @@ go_get(
         "huff0",
         "snappy",
         "zstd/...",
+        "zstd",
```

Comment:
Small regression here. @peterebden I wonder if this means it should be a v16 thing?

Reply:
I suppose it should be.

Does feel a bit weird to have to specify both too. You wouldn't normally have to do that with `go get`. I can't see a good way around though and the added flexibility might be worth the tradeoff (especially if we had a way to generate the `go_get` invocations)

Reply:
I found a way arolund it. ;P